server/server.js

```diff
@@ -38,7 +41,7 @@ app.use(
 );
 
 // Render React as View
-app.use(express.static(path.join(__dirname, "..", "client", "build")));
+// app.use(express.static(path.join(__dirname, "..", "client", "build")));
```

Comment:
Any concerns with this breaking production build?

Reply:
Honestly, I'm not sure. It works on my end. The idea is that the server serves JSON responses only, not views. I might be wrong though

Reply:
In production, the server serves both.  The client is only a separate app in development; in production, you compile the client app into some static assets, then serve the static assets with the middleware above. This probably breaks production :P

Reply:
Oh no, I am going to revert this too!

Is this how monorepos work? I only dealt with separate server and client repos, where the server responds with JSON and the client consumes it.

Reply:
Separate server and client repos are a common pattern in development, but you'll probably never see this used in production.